portfolio/README.md

```diff
@@ -5,4 +5,4 @@ command:
 
 ```bash
 mvn package appengine:run
-```
\ No newline at end of file
+```
```

Comment:
what changed on this line?

Reply:
Reverted Prettier since I can't get it to work properly.